sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -1399,15 +1399,25 @@ static bool _resolve_addr(const ipv6_addr_t *dst, gnrc_netif_t *netif,
     }
 #endif
 
+    /* don't do multicast address resolution on 6lo */
+    if (gnrc_netif_is_6ln(netif)) {
```

Comment:
Make sense. I moved this check. But I also kept the other check below:
```c
    if (!gnrc_netif_is_6ln(netif)) {
        _probe_nbr(entry, reset);
    }
```
because it is unclear for me whether that can be reached or not.

Reply:
I would even move it above the DEBUG() message.

Reply:
done!